src/main/java/thePackmaster/patches/lockonpack/ActualLockOnPatch.java

```diff
@@ -0,0 +1,40 @@
+package thePackmaster.patches.lockonpack;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch2;
+import com.megacrit.cardcrawl.actions.defect.LightningOrbEvokeAction;
+import com.megacrit.cardcrawl.actions.defect.LightningOrbPassiveAction;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.orbs.Lightning;
+import javassist.CannotCompileException;
+import javassist.expr.ExprEditor;
+import javassist.expr.MethodCall;
+import thePackmaster.powers.lockonpack.LightningRodPower;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+@SpirePatch2(clz = Lightning.class, method = "triggerPassiveEffect")
+@SpirePatch2(clz = LightningOrbPassiveAction.class, method = "update")
+@SpirePatch2(clz = LightningOrbEvokeAction.class, method = "update")
+public class ActualLockOnPatch {
+    public static AbstractMonster getLockedMonster(AbstractMonster originalTarget)
+    {
+        if (originalTarget.hasPower(LightningRodPower.POWER_ID)) return originalTarget;
+
+        ArrayList<AbstractMonster> lockedList = AbstractDungeon.getMonsters().monsters.stream().
```

Comment:
This also gets dead monsters, doesn't it? We have a method in Wiz to get the living enemies